lib/matplotlib/widgets.py

```diff
@@ -618,20 +617,18 @@ def clear(self):
         if self.drawon:
             self.ax.figure.canvas.draw()
 
-
     def get_status(self):
         """
         returns a tuple of the status (True/False) of all of the check buttons
         """
         return [l1.get_visible() for (l1, l2) in self.lines]
 
-
     def get_checked(self):
         """Returns a list of labels currently checked by user."""
 
-        return [l.get_text() for l, checked in zip(self.labels, self.get_status()) if
-                checked]
-
+        return [l.get_text() for l, box_checked in
+                zip(self.labels, self.get_status())
+                if box_checked ]
```

Comment:
`if box_checked ]` needs to become `if box_checked]`

That's the last PEP error. You're almost there... :tada:

Reply:
thanks, need to configure my IDE better, to not have to go through this so many times :)